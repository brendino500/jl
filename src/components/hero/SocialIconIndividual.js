import { makeStyles } from '@material-ui/core/styles'
import { IconButton } from '@material-ui/core'

const useStyles = makeStyles((theme) => ({
  icon: {
    color: '#EBE9EA',
    fontSize: '1.1em',
  },
}))

export default function SocialIconIndividual({ icon, url }) {
  const classes = useStyles()
  return (
    <>
      <IconButton
        href={url}
        target="_blank"
        rel="noreferrer"
        className={classes.icon}
      >
        {icon}
      </IconButton>
    </>
  )
}
