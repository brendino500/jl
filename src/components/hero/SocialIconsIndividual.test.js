import renderer from 'react-test-renderer'
import SocialIconIndividual from './SocialIconIndividual'

it('SocialIconIndividual matches snapshot', () => {
  const tree = renderer.create(<SocialIconIndividual />).toJSON()
  expect(tree).toMatchSnapshot()
})
