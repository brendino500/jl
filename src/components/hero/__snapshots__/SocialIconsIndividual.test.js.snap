// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`social media icons individual snapshot test renders correctly 1`] = `
<button
  className="MuiButtonBase-root MuiIconButton-root makeStyles-icon-1"
  disabled={false}
  onBlur={[Function]}
  onDragLeave={[Function]}
  onFocus={[Function]}
  onKeyDown={[Function]}
  onKeyUp={[Function]}
  onMouseDown={[Function]}
  onMouseLeave={[Function]}
  onMouseUp={[Function]}
  onTouchEnd={[Function]}
  onTouchMove={[Function]}
  onTouchStart={[Function]}
  rel="noreferrer"
  tabIndex={0}
  target="_blank"
  type="button"
>
  <span
    className="MuiIconButton-label"
  />
</button>
`;
